platforms/Dos/Makefile

```diff
@@ -29,8 +29,9 @@ CXXFLAGS := $(CFLAGS) -xds
 
 .PHONY: all clean
 
-all: CPPU.LIB CPPUX.LIB CPPU1.EXE CPPU2.EXE CPPU3.EXE CPPU4.EXE \
-    CPPU5.EXE CPPU6.EXE CPPU7.EXE
+all: CPPU.LIB CPPUX.LIB \
+     CPPU1.EXE CPPU2.EXE CPPU3.EXE \
+     CPPUX1.EXE CPPUX2.EXE CPPUX3.EXE CPPUX4.EXE
 
```

Comment:
Oh, okay. I guess I changed it at some point.

Still, I would suggest to make it two PRs - the first one for the essential fixes (like the PR I just submitted), and a second one for the name changes. It has been some time since I worked on this; initially I had namings like the ones you now suggest (and they make a lot of sense), then I changed it to simplify things. 

I am actually surprised that ALLTESTS.BAT runs just any .exe; this is not really good in my opinion!